deployments/Dockerfiles/documentation/Dockerfile

```diff
@@ -18,7 +18,7 @@ WORKDIR /build
 COPY docs/ /build/docs
 COPY mkdocs.yml /build
 
-RUN pip3 install bs4==0.0.1 markdown==3.3.4 click==8.0.2 mkdocs==1.3.0 markdown-include==0.6.0 mkdocs-material==7.1.10 mkdocs-macros-plugin==0.7.0 pymdown-extensions==8.2 Pygments==2.11.2
+RUN pip3 install bs4==0.0.1 markdown==3.3.7 click==8.0.2 mkdocs==1.4.2 markdown-include==0.8.0 mkdocs-material==9.0.4 mkdocs-macros-plugin==0.7.0 pymdown-extensions==9.9.1 Pygments==2.14.0
```

Comment:
Shouldn't we sue `tomte` for the doc dependencies as we do in `tox`

Reply:
I agree. You can consider this improvement in another ticker/issue, because this requires testing that it works, etc. This PR only fixes the versioning so that the used mkdocs features don't cause a fail in the release workflow.

Reply:
We don't have a release due right now so please do it on this PR

Reply:
fixed